wzdx/monitoring/monitor.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+import requests
+import time
+import logging
+from google.cloud import monitoring_v3
+
+WZDX_REST_ENDPOINT_PROD = os.getenv(
+    'WZDX_REST_ENDPOINT_PROD', 'https://data.cotrip.org/api/v1/wzdx?apiKey={api_key}')
+WZDX_REST_API_KEY_PROD = os.getenv(
+    'WZDX_REST_API_KEY_PROD', 'CT0E0KD-1S1MKYA-QSJ8WV7-045RH37')
```

Comment:
Is this API Key something we need to remove from public eye?

Reply:
Resolved in PR #51